Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -28,6 +16,12 @@ The features available from the external cluster will vary depending on the vers
 
 In order to configure an external Ceph cluster with Rook, we need to extract some information in order to connect to that cluster.
 
+### Prerequisites
```

Comment:
Move these prerequisites up to line 14. Currently this section is in the middle of running the commands on the source cluster. Also, change it to level 2 header.
```suggestion
## Prerequisites
```

Reply:
Sure